test/Npgsql.Tests/NotificationTests.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Npgsql.Tests
 {
+    [NonParallelizable]
```

Comment:
I don't mind this being NonParallelizable, but is there a specific problem with this test suite? Or are we leaving this for later?

Reply:
There are multiple calls `LISTEN notifytest` and `NOTIFY notifytest` from different tests. Definitely not impossible to fix, but probably better to do it a bit later.